samples/subsys/lorawan/class_a/sample.yaml

```diff
@@ -49,3 +49,8 @@ tests:
     filter: CONFIG_ENTROPY_HAS_DRIVER
     integration_platforms:
       - nucleo_wl55jc
+  sample.lorawan.class_a.multicast:
```

Comment:
This is no longer `class_a` only. I think it would make more sense as a separate sample.
This can be done in a later PR if you would prefer.

Reply:
I have a dedicated FUOTA sample ready, which will be added with the last step, the fragmented data transport service. In that step we can remove these bits from the class_a sample again, but we need this for now to make it compile in CI.